mirdata/datasets/beatles.py

```diff
@@ -36,17 +36,19 @@
     )
 }
 
+
+DATA = utils.LargeData("beatles_index.json")
+
+
 DOWNLOAD_INFO = """
         Unfortunately the audio files of the Beatles dataset are not available
         for download. If you have the Beatles dataset, place the contents into
         a folder called Beatles with the following structure:
             > Beatles/
                 > annotations/
                 > audio/
-        and copy the Beatles folder to {data_home}
-"""
-
-DATA = utils.LargeData("beatles_index.json")
+        and copy the Beatles folder to {}
+""".format(core.Dataset('beatles').data_home)
```

Comment:
it works, but it's making the code much slower - just to fill the `data_home` variable, we're instantiating the entire dataset object. is there a way to do this just inside `core`? like keeping the data home variable unset until it's used inside the Dataset object?

Reply:
I just realized also that this will always set `data_home` to the default, but if someone uses a custom one, it will still print the default